pyproject.toml

```diff
@@ -9,11 +9,11 @@
 
 [build-system]
 requires = [
-    "wheel",
+    "wheel<0.38.0",
     "setuptools<58.5",
```

Comment:
I'll check if we can bump this to `<60.0.0` by removing the pins in master.

Reply:
I'm actually not sure if we should bump to `59.5` or `60.0`. I think the former is safer, because it's not yet clear to me in which release `setuptools` will re-enable its vendored `distutils` copy.

Reply:
For the record: this got changed to `60.0`